<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pg_allplayers</class>
 <widget class="QWidget" name="pg_allplayers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>381</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>POPULATION</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);
font: 9pt &quot;Segoe Print&quot;;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe Print'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Start off with this distribution of players:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="slider" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>165</width>
       <height>73</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 9pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>COPYCAT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-20</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/image/hat_copycat.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignJustify</set>
      </property>
     </widget>
     <zorder>label_9</zorder>
     <zorder>label</zorder>
    </widget>
    <widget class="slider" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>165</width>
       <height>73</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 9pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>CHEATER</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/image/hat_cheater.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignJustify</set>
      </property>
     </widget>
     <zorder>label_10</zorder>
     <zorder>label_2</zorder>
    </widget>
    <widget class="slider" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>165</width>
       <height>73</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 9pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>COOPERATOR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>-10</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/image/hat_cooperator.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignJustify</set>
      </property>
     </widget>
     <zorder>label_11</zorder>
     <zorder>label_3</zorder>
    </widget>
    <widget class="slider" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>165</width>
       <height>73</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 9pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>GRUDGER</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>-10</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/image/hat_grudger.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignJustify</set>
      </property>
     </widget>
     <zorder>label_12</zorder>
     <zorder>label_4</zorder>
    </widget>
    <widget class="slider" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>165</width>
       <height>73</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 9pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>DETECTIVE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-20</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/image/hat_detective.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignJustify</set>
      </property>
     </widget>
     <zorder>label_13</zorder>
     <zorder>label_5</zorder>
    </widget>
    <widget class="slider" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>270</y>
       <width>165</width>
       <height>73</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 9pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>COPYKITTEN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-10</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/image/hat_copykitten.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignJustify</set>
      </property>
     </widget>
     <zorder>label_14</zorder>
     <zorder>label_6</zorder>
    </widget>
    <widget class="slider" name="widget_7" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>165</width>
       <height>73</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 9pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>SIMPLETON</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>0</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/image/hat_simpleton.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignJustify</set>
      </property>
     </widget>
     <zorder>label_15</zorder>
     <zorder>label_7</zorder>
    </widget>
    <widget class="slider" name="widget_8" native="true">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>370</y>
       <width>165</width>
       <height>73</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 9pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>RANDOM</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-20</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/image/hat_random.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignJustify</set>
      </property>
     </widget>
     <zorder>label_31</zorder>
     <zorder>label_8</zorder>
    </widget>
    <zorder>widget_2</zorder>
    <zorder>widget_3</zorder>
    <zorder>widget_4</zorder>
    <zorder>widget_5</zorder>
    <zorder>widget_6</zorder>
    <zorder>widget_7</zorder>
    <zorder>widget_8</zorder>
    <zorder>textEdit</zorder>
    <zorder>widget</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>PAYOFFS</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);
font: 9pt &quot;Segoe Print&quot;;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe Print'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;The payoffs in a one-on-one game are:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>-5</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>-140</x>
       <y>-20</y>
       <width>711</width>
       <height>511</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/payoffs.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>-5</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>-5</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>-5</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>210</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>-5</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_6">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>200</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>-5</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_7">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>310</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>-5</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_8">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>300</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>-5</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
    <zorder>label_32</zorder>
    <zorder>textEdit_3</zorder>
    <zorder>spinBox</zorder>
    <zorder>spinBox_2</zorder>
    <zorder>spinBox_3</zorder>
    <zorder>spinBox_4</zorder>
    <zorder>spinBox_5</zorder>
    <zorder>spinBox_6</zorder>
    <zorder>spinBox_7</zorder>
    <zorder>spinBox_8</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>RULES</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);
font: 9pt &quot;Segoe Print&quot;;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe Print'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Play            rounds per match:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>361</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);
font: 9pt &quot;Segoe Print&quot;;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe Print'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;After each tournament, eliminate the bottom          players &amp;amp; reproduce the top             players:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>361</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);
font: 9pt &quot;Segoe Print&quot;;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe Print'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;During each round, there's a            % chance a player makes a mistake:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_9">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_11">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>290</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_12">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_13">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>69</y>
     <width>160</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Step</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>slider</class>
   <extends>QWidget</extends>
   <header>slider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
